arch/avr/src/avr/up_createstack.c

```diff
@@ -118,9 +137,42 @@ int up_create_stack(FAR struct tcb_s *tcb, size_t stack_size, uint8_t ttype)
     {
       /* Allocate the stack.  If DEBUG is enabled (but not stack debug),
        * then create a zeroed stack to make stack dumps easier to trace.
+       * If TLS is enabled, then we must allocate aligned stacks.
        */
+#ifdef CONFIG_TLS_ALIGNED
```

Comment:
Is there a blank line missing here?

Reply:
There is another #ifdef just after, so no blank line needed.